README.md

```diff
@@ -44,21 +44,23 @@ The repository has a `.vscode` folder that contains `settings.json` and `extensi
 
 3. Give vercel the nessecary permissions, deploy the projects and voila the deployment is done.
 
-## How to enable email auth in Supabase
+## How to set up Supabase
 
 1. Sign up on Supabase [here](https://supabase.com/).
 
-2. Create a new Project inside Supabase
+2. Create a new Project inside Supabase.
 
-3. Go to settings -> API and copy the Project `URL` and `Anon/Public`
+3. Go to settings -> API and copy the Project `URL` and `Anon/Public`.
 
-4. Paste the values in the .env.template at `NEXT_PUBLIC_SUPABASE_URL` for the URL and `NEXT_PUBLIC_SUPABASE_ANON_KEY` for the anon.
+4. Paste the values in the `.env.template` at `NEXT_PUBLIC_SUPABASE_URL` for the URL and `NEXT_PUBLIC_SUPABASE_ANON_KEY` for the anon.
 
-5. Go to the [SQL Editor](https://app.supabase.com/project/_/sql) Page inside the Supabase Dashboard
+5. Change the name of `.env.template` to `.env.local`.
```

Comment:
I think we should encourage people to create a new file called `.env.local` instead. Renaming `.env.template` is essentially deleting it, and people might accidentally commit it.